src/components/Codeblock.tsx

```diff
@@ -276,6 +276,7 @@ const Codeblock = ({
               style={style}
               className={className}
               pt={hasTopBar ? "2.75rem" : 6}
+              pb={"1.5rem"}
```

Comment:
@sandeepV2 With our design system, we use spacing tokens where `1` = `0.25rem` = `4px`... let's use the `6` token here instead:

```suggestion
              pb={6}
```

Reply:
@wackerow Yes i see that 6 token in the padding-top styling too.
`pt={hasTopBar ? "2.75rem" : 6}`

I tried to find where we define the spacing tokens where `1 = 0.25rem =4px` , is it in spacing.ts ?

[https://github.com/ethereum/ethereum-org-website/blob/dev/src/%40chakra-ui/foundations/spacing.ts#L2](https://github.com/ethereum/ethereum-org-website/blob/dev/src/%40chakra-ui/foundations/spacing.ts#L2)

> const spacing = {
  7.5: "1.875rem",
  10.5: "2.625rem",
  19: "4.75rem", // Nav height
}

is it hiding in here (`1.875rem / 7.5 = 0.25rem `) ?